### PR TITLE
Add methods parsing query parameters and storing tokens to a localsto…

### PR DESCRIPTION
**Pull request by shskwmt**
Add following methods to CognitoUtil

**1. storeTokenFromQueryString**

This method parses a query string of given callback URI and retrieves access_token, id_token and expires_in. Then it stores these values to a localstorage through `storeToken` method.

**2. storeToken**

This method stores access_token, id_token and expires_in in localstorage.

### CODE REVIEW

File: MIT-LICENSE [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Copyright 2020
```
Comment (qpSHiNqp) [inline MIT-LICENSE:1]:
```suggestion
Copyright 2020 ONE CAREER, Inc.
```

**Review by qpSHiNqp — CHANGES_REQUESTED** [review]
Copyright のあとは著作権者書かないといけないと思います。
`const` な変数には方宣言入れないって感じですよね？

**Comment by shskwmt** [comment]
Thank you for your comments.

> Copyright のあとは著作権者書かないといけないと思います。

Oh, I forgot it. I'll do so!

> `const` な変数には方宣言入れないって感じですよね？

I think it is better to omit a type annotation when it can be inferred.

**Review by qpSHiNqp — APPROVED** [review]
Thank you, fixes are confirmed.

**Pull request merged**